### PR TITLE
[orc8r][lte][cwf] Split support of relay_enabled for hss and gx_gy on gateway

### PR DESCRIPTION
**Pull request by andreilee**
Signed-off-by: Andrei Lee <andreilee@fb.com>

## Summary

Federation support is being split between gx_gy and hss. In terms of configuration, the `relay_enabled` flag is being split to support this, with new `gx_gy_relay_enabled` and `hss_relay_enabled` flags.

This first PR adds support for this on the access gateway, and is backwards compatible, but will use the new flags. Later PRs will modify the orc8r side and API to replace `relay_enabled`, and remove it on the gateway.

**Specific changes:**
- proto messages modified to mark `relay_enabled` as `DEPRECATED`, but they are not disabled fields
- proto messages to add `hss_relay_enabled` and `gx_gy_relay_enabled`, but these fields are only read in the gateway
- cwf gateway integ tests modified for relay split
- `sessiond` internally renames `relay_enabled` to `gx_gy_relay_enabled`
- `mme` internally renames `relay_enabled` to `hss_relay_enabled`

## Test Plan

- CI checks

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by themarwhal — APPROVED** [review]
SessionD changes lgtm

File: lte/gateway/c/oai/include/pgw_config.h [hunk 1]
```diff
@@ -115,7 +115,7 @@ typedef struct pgw_config_s {
 
   bool force_push_pco;
   uint16_t ue_mtu;
-  bool relay_enabled;
+  bool hss_relay_enabled;
```
Comment (ssanadhya) [inline lte/gateway/c/oai/include/pgw_config.h]: Remove this as relay_enabled is not used in the SPGW task any more. All session messages are sent to SessionD.

File: lte/gateway/c/oai/tasks/sgw/pgw_config.c [hunk 2]
```diff
@@ -440,10 +440,10 @@ int pgw_config_parse_file(pgw_config_t* config_pP) {
     if (config_setting_lookup_string(
             setting_pgw, PGW_RELAY_ENABLED, (const char**) &astring)) {
       if (strcasecmp(astring, "yes") == 0) {
-        config_pP->relay_enabled = true;
+        config_pP->hss_relay_enabled = true;
```
Comment (ssanadhya) [inline lte/gateway/c/oai/tasks/sgw/pgw_config.c]: Remove this config as it is not used anywhere.

**Review by ssanadhya — CHANGES_REQUESTED** [review]

**Review by uri200 — APPROVED** [review]

File: lte/cloud/go/services/cellular/protos/cellular_service.proto [hunk 3]
```diff
@@ -137,6 +137,7 @@ message NetworkEPCConfig {
     }
     map<string, SubscriptionProfile> sub_profiles = 6;
     string default_rule_id = 7;
+    // DEPRECATED
```
Comment (xjtian) [inline lte/cloud/go/services/cellular/protos/cellular_service.proto:140]: This entire package is deprecated actually (there are no usages of it and there shouldn't be). Looks like I just forgot to clean this up with the configurator migration a long time ago. You can revert this change, I'll rm this package myself

**Review by xjtian — APPROVED** [review]

**Review by ssanadhya — APPROVED** [review]
Changes to MME service look good to me.

**Pull request merged**